README.md

```diff
@@ -1,27 +1,42 @@
+<div align="center">
 
-## Orbot: Android Onion Routing Robot
+<img width="" src="./app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png" alt="Orbot" align="center"/>
 
-[![Build Status](https://app.bitrise.io/app/0e76c31b8e7e1801/status.svg?token=S2weJXueO3AvrDUrrd85SA&branch=master)](https://app.bitrise.io/app/0e76c31b8e7e1801) (see Orbot's [CI docs](./docs/CI.md))
+# Orbot
+
+### Android Onion Routing Robot
+
+[![Play Downloads](https://img.shields.io/github/downloads/guardianproject/orbot/total)](https://play.google.com/store/apps/details?id=org.torproject.android)
```

Comment:
Any way we can host the graphics ourselves, so that we aren't embedding images from third-party websites?

Reply:
I changed it so that all the badges are hosted in the repo, and we can update them periodically.